src/internal/m365/collection/drive/collections_tree_test.go

```diff
@@ -958,13 +980,79 @@ func (suite *CollectionsTreeUnitSuite) TestCollections_PopulateTree_singleDelta(
 				sizeBytes:      0,
 				treeContainsFolderIDs: []string{
 					rootID,
-					idx(folder, "parent"),
+					folderID("parent"),
 				},
 				treeContainsTombstoneIDs: []string{
-					id(folder),
+					folderID(),
+				},
+				treeContainsFileIDsWithParent: map[string]string{
+					fileID(): folderID(),
+				},
+			},
+		},
+		{
+			name: "delete->create folder with previous path",
```

Comment:
are these new tests from a later PR? It's not clear from the test right now that there's previous paths associated with the deleted folder so I'm a little confused

Reply:
Ah, I renamed the test in a following PR.  It's kind of imaginging an nth delta enumeration where a prior delta introduced the file, then the next delta has a delete/recover pair of operations.